pyensembl/database.py

```diff
@@ -223,7 +224,7 @@ def _connect_if_exists(self):
     @property
     def connection(self):
         """
-        Return the sqlite3 database for this Ensembl release
+        Return the sqlite3 database for this Genomic database
```

Comment:
"Genomic" is unexpectedly capitalized (or maybe should be "genome annotation" or "genome annotation dataset")